README.md

```diff
@@ -19,8 +19,8 @@ If you need help using or installing Node.js, please use the
 
 ## Resources for Newcomers
 
-* [Website](https://nodejs.org/en/)
-* [Contributing to the project](./CONTRIBUTING.md)
+* Website: [nodejs.org](https://nodejs.org/en/)
```

Comment:
I know this wasn't part of this PR, but does anyone know why we are sending everyone to `/en`?